util/settings/settings.go

```diff
@@ -318,6 +318,20 @@ type RepositoryCredentials struct {
 	Type string `json:"type,omitempty"`
 }
 
+// DeepLink structure
+type DeepLink struct {
```

Comment:
Please add deep links sample configurations to https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/argocd-cm.yaml